style.css

```diff
@@ -1,246 +1,451 @@
+  grid-template-rows: auto;
+  margin: 5% 0%;
+}
+
+.desk {
   display: grid;
-  grid-template-rows: auto auto auto auto;
+  grid-template-columns: 100% 100%;
 }
 
 .tonic {
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-  padding: 1.3rem;
-  margin: 44px 5%;
-  width: 90%;
-  background: white;
-  border-radius: 16px;
-}
-
-.can {
-  font-size: 13px;
-  font-weight: 500;
+  display: grid;
+  grid-template-columns: 49% 49%;
+  background-color: white;
+  margin: 5% 10%;
+  border-radius: 10px;
+  column-gap: 1%;
 }
 
-.back {
-  color: #7a869a;
-  font-size: 13px;
+.tonic2 {
+  padding: 5% 3%;
 }
 
-.knop {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  padding: 0;
-  gap: 12px;
-  width: 247px;
-  margin: 15px 0;
+.tonic3 {
+  padding-left: 5%;
 }
 
-h1 {
-  padding-right: 50px;
+.nature {
+  padding: 3%;
+}
+
+.title {
   font-size: 40px;
-  margin-bottom: 12px;
+  font-weight: 700;
   line-height: 52px;
-  font-style: bold;
-  color: rgba(23, 43, 77, 1);
+  text-align: left;
+  color: #091e42;
 }
 
-.ulto {
-  margin-top: 35px;
+@media screen and (min-width: 768px) {
+  .title {
+    font-size: 22px;
+    font-weight: 700;
+    line-height: 32px;
+    text-align: left;
+    color: #091e42;
+  }
+
+  .tonic3 {
+    padding-left: 10%;
+  }
+
+  .tonic4 {
+    padding-right: 2%;
+  }
+}
+
+.canopy {
   display: flex;
-  flex-direction: row;
-  align-items: center;
-  padding: 0;
-  gap: 12px;
-  width: 236px;
-  height: 40px;
-  order: 1;
-  flex-grow: 0;
+  margin: 3% 0 5% 0;
 }
 
-.ulto > li {
-  font-style: normal;
-  font-weight: 600;
-  font-size: 13px;
-  line-height: 16px;
-  margin-left: 10px;
-  color: #7a869a;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
+.canopy > h6 {
+  font-size: 18px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #344563;
+  margin-right: 2%;
+}
+
+h5 img {
+  margin-bottom: -10px;
+}
+
+@media screen and (min-width: 768px) {
+  .canopy > h6 {
+    font-size: 10px;
+    font-weight: 500;
+    line-height: 6px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #344563;
+    margin-right: 2%;
+  }
+
+  .canopy > img {
+    width: 2%;
+    height: 2%;
+  }
+}
+
+.canopy > img {
+  margin-top: 0.5%;
+}
+
+.back {
+  font-size: 18px;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+  color: #6b778c;
+  margin: 0 2%;
+}
+
+@media screen and (min-width: 768px) {
+  .back {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 8px;
+    text-align: left;
+    color: #6b778c;
+    margin: 0 2%;
+  }
 }
 
 .daily {
-  font-style: normal;
+  font-size: 16px;
   font-weight: 400;
-  font-size: 15px;
   line-height: 24px;
+  text-align: left;
   color: #344563;
-  flex: none;
-  order: 1;
-  flex-grow: 0;
 }
 
-.flex-box {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  padding: 0;
-  gap: 8px;
-  line-height: 24px;
-  width: 203px;
-  height: 50px;
+@media screen and (min-width: 768px) {
```

Comment:
Instead of repeating `@media screen and (min-width: 768px)` and `@media screen and (min-width: 767px)` several times in your code, kindly consider making your applying the DRY (Don't Repeat Yourself) principle by stating it only once and putting all the other codes that fall within the `min-width: 768px` inside.